docs/deploying/windows.md

```diff
@@ -12,11 +9,32 @@ There are 4 primary steps to deploying and running hubot on a Windows machine:
   * setting up environment variables for hubot
   * a way to start hubot, start it up if it crashes, and restart it when code updates
 
+# PowerShell Module
+
+A PowerShell Module called `PoshHubot` is available which will allow you to install and configure Hubot for Windows with a few commands.
+
+If you have PowerShell v5 installed you can simple run the following command to install the module:
+
+```
+Install-Module -Name PoshHubot
+```
+Details about using Hubot on Windows with PowerShell can be found in this blog post: https://hodgkins.io/chatops-on-windows-with-hubot-and-powershell
```

Comment:
Is there any information on that blog post that could be included here?  Or maybe it could be in the PoshHubot repo at https://github.com/MattHodge/PoshHubot?